src/converter.ts

```diff
@@ -158,6 +159,10 @@ async function createConverter(converterType: string): Promise<AbstractConverter
             console.log("[i] Processing file using XTB converter");
             converter = new XtbConverter(securityService);
             break;
+        case "directa":
+            console.log("[i] Processing file using Directa converter, this is an experimental converter!");
```

Comment:
You can remove the part of it being experimental if you're satisfied with it and are ready to merge. I'm OK with this not being here in the version in main.